### ISSUE TITLE
Minor visual bug: Inline math is slightly smaller than surrounding text

### ISSUE THREAD

**Issue by jxxe** [notaapp/talk#351]
This is an intentional visual design decision, but it messes up spacing for tables.

<img width="253" alt="image" src="https://user-images.githubusercontent.com/64276359/214956277-832426b5-98e7-41f3-8a7e-47555d91c2bb.png">

```
| $n$ | First values |
|-----|--------------|
| 1   | 1, 2, 3      |
| 2   | 2, 3, 4      |
```

**Comment by astoilkov** [comment]
Is this from 0.38 or 0.39?

**Comment by jxxe** [comment]
0.38, it's been fixed as of 0.39.1
